### PR TITLE
Updating meta.yaml to make it pure template

### PR DESCRIPTION
**Pull request by jsignell**
Getting rid of dependencies for test and nodejs requirement. If nodejs is needed it should be moved to setup.py

### CODE REVIEW

**Comment by jbednar** [comment]
Sounds good to me!

**Comment by philippjfr** [comment]
It is indeed required, doesn't need to be pinned though. Would be good to add it to ``extras_require['build']`` in this PR.

File: setup.py [hunk 1]
```diff
@@ -28,6 +28,46 @@ def get_setup_version(reponame):
         print("WARNING: param>=1.6.0 unavailable. If you are installing a package, this warning can safely be ignored. If you are creating a package or otherwise operating in a git repository, you should install param>=1.6.0.")
         return json.load(open(version_file_path, 'r'))['version_string']
 
+def build_custom_models():
```
Comment (jsignell) [inline setup.py]: I moved this up in the file to make the setup.py easier to read.

Comment (jbednar) [reply setup.py]: Ok

Comment (jbednar) [reply setup.py]: Merge at your leisure!

**Pull request merged**